### ISSUE TITLE
Cannot read property 'unsubscribe' of undefined

### ISSUE THREAD

**Issue by shelang08** [ngx-formly/ngx-formly#2781]
Got error when create a custome tree select template with NzTreeSelectModule

Template:
 <nz-tree-select
    [nzNodes]="to.options"
    nzShowSearch
    [(ngModel)]="value"
  >
  </nz-tree-select>

this.selectionChangeSubscription is undefined.
```
ngOnDestroy() {
        this.isDestroy = true;
        this.closeDropDown();
        this.selectionChangeSubscription.unsubscribe();
        this.destroy$.next();
        this.destroy$.complete();
        this.focusChangeSubscription.unsubscribe();
  }
```

- Angular version: 11.2.5
- Formly version: 6.0.0.next.1
- ng-zorro-antd version: 11.2.0

**Comment by aitboudad** [comment]
That's an issue of `ng-zorro-antd`, I've sent a fix https://github.com/NG-ZORRO/ng-zorro-antd/pull/6574 where it should be part of their next release.

**Comment by zzhimin** [comment]
> That's an issue of `ng-zorro-antd`, I've sent a fix [NG-ZORRO/ng-zorro-antd#6574](https://github.com/NG-ZORRO/ng-zorro-antd/pull/6574) where it should be part of their next release.

you can also use this library：https://github.com/zzhimin/ngx-ng-zorro-formly
